cartaya/1/style.css

```diff
@@ -0,0 +1,32 @@
+body {
+    margin: 0;
+    font-family: sans-serif;
+}
+section {
+	height: 75vh;
+	background-color: #FFA600;
+}
+a {
+    font-size: 48px;
+    display: inline-block;
+    text-decoration: none;
+    border-bottom: 5px solid rgba(255, 255, 255, 0.692); 
+    color: rgba(255, 255, 255, 0.877);

```

Comment:
You can define the color in the element container, this is an Inherited property you don’t need specify in each element inside container